src/hint-core.scss

```diff
@@ -67,6 +67,7 @@
 		font-family: $hintFontFamily;
 		line-height: $hintFontSize; // Vertical centering.
 		white-space: nowrap; // Prevent breaking to new line.
+		word-wrap: break-word; // Ensure long words do not overflow.
```

Comment:
I suppose you are using one of the width limiting classes in your screenshot? In that case, `word-wrap` property should only be applied for those 3 classes because otherwise it won't make any sense as words won't break at all.